src/pages/projectCreate/components/ProjectTagsCreateEdit.tsx

```diff
@@ -162,19 +162,19 @@ export const ProjectTagsCreateEdit = ({
     )
   }
 
-  const isDisabled = tags.length >= 3
+  const isDisabled = tags.length >= 4
   const showAddTag = !tagOptions.some((tag) =>
     tag.label.toLowerCase().includes(inputValue.toLowerCase()),
   )
-  const disableShowAddTag = inputValue.length < 3 || createLoading
+  const disableShowAddTag = inputValue.length < 4 || createLoading
```

Comment:
maybe add a const MAX_TAGS_ALLOWED, value, that'll be easy to see and use. We use all caps for simple constant cases like this one.

Reply:
Updated